File: src/main/java/com/devdeolho/cleanarch/CleanArchApplication.java
package com.devdeolho.cleanarch;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;


@EnableFeignClients
@SpringBootApplication
public class CleanArchApplication {

	public static void main(String[] args) {
		SpringApplication.run(CleanArchApplication.class, args);
	}

}

File: src/main/kotlin/com/devdeolho/cleanarch/CleanarchApplication.kt
package com.devdeolho.cleanarch

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.openfeign.EnableFeignClients

@EnableFeignClients
@SpringBootApplication
class CleanarchApplication

fun main(args: Array<String>) {
	runApplication<CleanarchApplication>(*args)
}
